lib/ui/screens/present_signed_document_screen.dart

```diff
@@ -153,7 +153,7 @@ class _Body extends StatelessWidget {
       PresentSignedDocumentLoadingState _ => const LoadingContent(),
       PresentSignedDocumentErrorState _ => _SuccessContent(
           file: null,
-          onShareFileRequested: null,
+          onShareFileRequested: onShareFileRequested,

```

Comment:
Sharovať sa bude dať, aj keď sa podpíše dokument z externého systému? (to by sa asi nemalo, ale len sa pýtam, možno to funguje tak, ako chceme)

Reply:
Pozriem.

Reply:
Uprave. vid commit 725f62877fabec5636bfdf40bfe8bef6ab0fbd8e